cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/IndexTest.scala

```diff
@@ -87,6 +87,15 @@ Create an index on the label `Person` and property `age` with the name `index_na
 ###assertion=create-index
 //
 
+CREATE INDEX FOR (p:Person) ON (p.surname)
+OPTIONS {indexProvider: 'native-btree-1.0', indexConfig: {`spatial.cartesian.min`: [-100.0, -100.0], `spatial.cartesian.max`: [100.0, 100.0]}}
+###
+
+Create an index on the label `Person` and property `surname` with the index provider `native-btree-1.0` and updated `spatial.cartesian` settings.
```

Comment:
Updated? So, it stays afterward?

Reply:
I think it is just for that index, but I don't really know how the config settings work.

@burqen Can you confirm that it is for this index only?

Reply:
The index config is for the index (also for index backing a constraint) and I would say 'updated' is not the right word. It indicates to me that there already was some configuration for and existing index that we change, but the index is created with the given configuration.
Defining indexConfig for a single index creation does not change any default values.

I hope I interpreted your questions correctly ☺️